resources/build/build-utils.md

```diff
@@ -388,6 +408,35 @@ script. If not specified, dependencies will be built for all actions on all
 targets. Either `action` or `:target` may be omitted, and multiple actions and
 targets may be specified, space separated.
 
+--------------------------------------------------------------------------------
+
+## `builder_describe_internal_dependency` function
+
+Define a local dependency between one action:target and another.
+
+### Usage
+
+```bash
+builder_describe_internal_dependency action:target depaction:deptarget ...
+```
+
+### Parameters
+  * **action:target**:        The action and target that has a dependency
+  * **depaction:deptarget**:  The dependency action and target
+
+### Example
+
+```bash
+builder_describe_internal_dependency \
+  mac:build mac-x86_64:build \
+  mac:build mac-arm64:build
+```
+
+**Note:** actions and targets must be fully specified, and this _must_ be called
+before either builder_describe_outputs or builder_parse in order for
+dependencies to be resolved.
+
```

Comment:
Do the new colors also get added to https://github.com/keymanapp/keyman/blob/1bc31c8792b460233ab015c59567ef2d5e6bf966/resources/build/build-utils.md#formatting-variables ?

Reply:
Yes, may be a good idea, perhaps under a new `builder_echo` topic